src/main/java/com/fitwsarah/fitwsarah/appointmentsubdomain/datalayer/AppointmentRepository.java

```diff
@@ -8,8 +8,10 @@
 @Embeddable
 public interface AppointmentRepository extends JpaRepository<Appointment, Integer> {
     Appointment findAppointmentsByAppointmentIdentifier_AppointmentId( String appointmentId);
-    List<Appointment> findAppointmentByAccountId(String accountId);
+
     List<Appointment> findAllAppointmentsByAppointmentIdentifier_AppointmentIdStartingWith(String appointmentId);
     List<Appointment> findAllAppointmentsByStatus(Status status);
-    List<Appointment> findAllAppointmentsByAccountIdStartingWith(String accountId);
+    List<Appointment> findAllAppointmentsByUserIdStartingWith(String accountId);
```

Comment:
Here it looks like youre trying to find all appointments by user id starting with accountId, but isn't account id a uuid, so i don't understand how it works, if you could explain it.

Reply:
The accountId is a simply a string that will take any kind of string inside, it could be in the form of an userId or a uuid. I will change the name. Sorry for the confusion.